DOCS/license-inclusion-principles.md

```diff
@@ -7,42 +7,41 @@ Determining whether a candidate license should be included on the SPDX License L
-* The license steward supports or is at least aware of and does not oppose its submission to the SPDX License List.
+  * [Open Definition](http://opendefinition.org/od/2.1/en/) from the Open Knowledge Foundation
+  * [Free Cultural Works definition](https://freedomdefined.org/Definition) from Freedom Defined.org
+2. The license is structured to be generally usable by anyone. It is not a "vanity" license or specific to one project, consortium or corporation.
+3. The license has actual, substantial use such that it is likely to be encountered. Substantial use may be demonstrated via use in many projects, or in one or a few highly significant projects. For new licenses, there is definitive plans for the license to be used in one or a few significant projects.
+4. If the license does not substantively comply with one of the above open source definitions, then the license is primarily intended for free distribution of content (including, in the case of software, its source code) with limited restrictions, and meets other factors listed here.
+5. The license steward supports or is at least aware of and does not oppose its submission to the SPDX License List.
 
-The SPDX legal team will also consider any information considered relevant by the license steward or open source community.
+The SPDX legal team will also consider any other information considered relevant by the license steward or open source community.
 
 Comments on license inclusion will be noted in the Github issue for the license (or reference to meeting minutes, as necessary). Some license requests may be decided solely via comments in the Github issue and some may involve discussion on the bi-weekly legal call. Thus it is imperative that requestors join the mailing list and calls to fully participate.
 
-*last updated Feb 2020*
+*last updated Mar 2020*
 
-## Historical Background
-The first beta version of the SPDX License List was published in August 2010 and had approximately one hundred licenses on it. The initial set of licenses was included based on informal discussion and consensus on the SPDX working group calls and email list. Although various "guidelines" were identified in regards to which licenses to include or not during this time, formal guidelines were not recorded beyond the meeting minutes. We had to start somewhere, and the obvious was the easy beginning point. Decisions or guidelines that evolved out of discussion or by implication included the following:
-* Include commonly found open source licenses. Although discussion was not explicit in regards to how to define an "open source license," this was always an implicit guiding principle
-* Include all OSI approved licenses, both current and those approved but now deprecated. The rationale being that once OSI-approved, always OSI-approved and deprecated licenses still appear "in the wild"
+# Historical Background
```

Comment:
Per my earlier comment on the PR, I wonder whether this is part of the guidelines, to be read with them, or could be moved to a separate document.

Reply:
see comment above